.circleci/ci_package.sh

```diff
@@ -15,38 +15,40 @@ fi
 # Install my fork of RAMP- need to merge it first
 pip install git+https://github.com/mnunberg/RAMP@print_version --upgrade
```

Comment:
I know you did not changed it here but what is this version and why its not on master ?

Reply:
I don't have push access to that repo. The change I'm talking about with different shards though, I did locally. Also, the more relevant change is https://github.com/RedisLabsModules/RediSearch/pull/471. This PR just adds some logging fixes in the API